cli/kata-check_amd64.go

```diff
@@ -116,21 +116,24 @@ func setCPUtype(hypervisorType vc.HypervisorType) error {
 			}
 			archRequiredKernelModules = map[string]kernelModule{
 				kernelModkvm: {
-					desc: msgKernelVM,
+					desc:     msgKernelVM,
+					required: true,
 				},
 				kernelModkvmintel: {
 					desc:       "Intel KVM",
 					parameters: kvmIntelParams,
+					required:   true,
 				},
 				kernelModvhost: {
-					desc: msgKernelVirtio,
+					desc:     msgKernelVirtio,
+					required: true,
 				},
 				kernelModvhostnet: {
-					desc: msgKernelVirtioNet,
+					desc:     msgKernelVirtioNet,
+					required: true,
 				},
 				kernelModvhostvsock: {
-					desc:     msgKernelVirtioVhostVsock,
-					required: false,
```

Comment:
missing `required: true,` ?

Reply:
vosck is not required, and by default required is false, so I just removed the line as it's redundant.

Reply:
I didn't know that by default it's `false`, can we keep it explicit?

Reply:
That's how [Go works](https://tour.golang.org/basics/12), but I am happy to set it explicitly to `false` if this gives more clarity.

Reply:
thanks @marcov